x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilege.java

```diff
@@ -49,7 +54,8 @@
     private static final Automaton MONITOR_WATCHER_AUTOMATON = patterns("cluster:monitor/xpack/watcher/*");
     private static final Automaton MONITOR_ROLLUP_AUTOMATON = patterns("cluster:monitor/xpack/rollup/*");
     private static final Automaton ALL_CLUSTER_AUTOMATON = patterns("cluster:*", "indices:admin/template/*");
-    private static final Automaton MANAGE_AUTOMATON = minusAndMinimize(ALL_CLUSTER_AUTOMATON, MANAGE_SECURITY_AUTOMATON);
+    private static final Automaton MANAGE_AUTOMATON = minusAndMinimize(minusAndMinimize(ALL_CLUSTER_AUTOMATON, MANAGE_SECURITY_AUTOMATON),
+            DELEGATE_PKI_AUTOMATON);
```

Comment:
I went with your suggestion. Thanks for the details!

One idea I had in mind was to have this cluster action named completely different, for example `cluster:authenticate/pki/delegate`. The reason is that it doesn't fit under the `admin` namespace (the user doing delegation could very well not be an "admin"), and it avoids possible misconfiguration when users define privileges by patterns. But I backtracked because of a lack of a good name. If you feel this is a good idea and have a name suggestion, we could give it a try.

Reply:
I don't have a good suggestion. I think putting it under admin is OK (but I agree with your attempts to find an even better place for it).